pygmt/tests/test_logo.py

```diff
@@ -1,34 +1,26 @@
 """
 Tests for fig.logo.
 """
+import pytest
 from pygmt import Figure
-from pygmt.helpers.testing import check_figures_equal
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_logo():
     """
     Plot a GMT logo of a 2 inch width as a stand-alone plot.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    # Use single-character arguments for the reference image
-    fig_ref.logo(D="x0/0+w2i")
-    fig_test.logo(position="x0/0+w2i")
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.logo(position="x0/0+w2i")
```

Comment:
`Figure.logo()` doesn't require any parameters:
```suggestion
    fig.logo()
```